### DISCUSSION TITLE
Show previous chat messages in chat widget.

### DISCUSSION THREAD

**Discussion by QomariSanjaya** [LiveHelperChat/livehelperchat#1952] [Q&A]
If I checked both _"Reopen chat functionality enabled"_ and _"Allow user to reopen closed chats?"_ the setting _"Show previous chat messages in chat widget."_ seems not working because there is no previous message shown. Or I miss another hidden setting?

**Comment by PeopleInside** [comment]
Are you able to reproduce on the demo?
https://livehelperchat.com/demo-12c.html

If so you can provide steps to reproduce and maybe a short video.
If you think you found a BUG you can report it as issue.

https://livehelperchat.com/how-to-report-a-bug-properly-310a.html
